### PR TITLE
LPS-144178 Rename 'attributes' to 'search context attributes'

### PR DESCRIPTION
**Pull request by oliv-yu**
https://issues.liferay.com/browse/LPS-144178 - "Attributes" is renamed to "Search Context Attributes" on the Blueprint editor Preview sidebar

Some simple renaming and updating the language keys! Also added in those changes to clarify the organization of sidebar components, and make PreviewAttributesModal more reusable by passing in title, description, and the child trigger.

### CODE REVIEW

File: modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/EditSXPBlueprintForm.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ import {
 	validateNumberRange,
 	validateRequired,
 } from '../utils/validation';
-import AddSXPElementSidebar from './AddSXPElementSidebar';
+import AddSXPElementSidebar from './add_sxp_element_sidebar';
```
Comment (thektan) [inline modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/EditSXPBlueprintForm.js:52]: This import won't resolve in portal. It'll resolve in storybook, but in portal we use our own module resolver so we need the `/index` added to this.

File: modules/dxp/apps/search-experiences/search-experiences-web/test/js/edit_sxp_blueprint/add_sxp_element_sidebar/index.js [hunk 2]
```diff
@@ -12,8 +12,8 @@
 import {fireEvent, render} from '@testing-library/react';
 import React from 'react';
 
-import AddSXPElementSidebar from '../../../src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/AddSXPElementSidebar';
-import {QUERY_SXP_ELEMENTS} from '../mocks/data';
+import AddSXPElementSidebar from '../../../../src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/add_sxp_element_sidebar';
```
Comment (thektan) [inline modules/dxp/apps/search-experiences/search-experiences-web/test/js/edit_sxp_blueprint/add_sxp_element_sidebar/index.js:15]: Let's also add the `/index` to this path to keep the paths consistent even though it isn't necessary here.

**Review by thektan — CHANGES_REQUESTED** [review]
@oliv-yu Thanks for organizing those files and improving the attributes modal reusability!

Just update that import path since it causes the entire edit page to not render which is why so many of the tests are failing.

File: modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/EditSXPBlueprintForm.js (hunk 1 above)
Comment (oliv-yu) [reply modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/edit_sxp_blueprint/EditSXPBlueprintForm.js]: Added, thanks for letting me know!

**Comment by oliv-yu** [comment]
ci:test:search

**Comment by thektan** [comment]
ci:test:sf

**Review by thektan — APPROVED** [review]

**Comment by joshchong** [comment]
Resent with test fix here: https://github.com/liferay-search/liferay-portal/pull/303

**Pull request closed without merging**